website/server/controllers/api-v3/tasks.js

```diff
@@ -578,11 +765,15 @@ api.scoreCheckListItem = {
  * @apiParam {String} taskId The task _id or alias
  * @apiParam {UUID} itemId The checklist item _id
  *
+ * @apiParam (body) {string} text The text that will replace the current checkitem text.
+ *
+ * @apiParamExample {json} Example body:
+ * {"text":"Czech 1"}
+ *
  * @apiSuccess {Object} data The updated task
  *
  * @apiUse TaskNotFound
  * @apiUse ChecklistNotFound
- * @apiUse ChallengeNotFound
```

Comment:
To be honest, this raises more questions for me... if a task was linked to a challenge and that challenge no longer exists, that means there is no way to delete that task. Wouldn't it make more sense to allow the task to be deleted instead of throwing an error that can't be corrected? 

I will add it back in, but I am adding a note to try and explain it better because I do not think that the error message makes sense.